modules/btcpay/src/Server/WebhookHandler.php

```diff
@@ -46,6 +53,8 @@ public function process(Request $request): void
 
 		// Check if we received an event
 		if (!\array_key_exists('type', $data) || !\array_key_exists('invoiceId', $data)) {
+			\PrestaShopLogger::addLog(\sprintf('[INFO] No event or invoice ID received: %s', \json_encode($data, \JSON_THROW_ON_ERROR)));
```

Comment:
I may do this too or in a more limited way only for valid webhook events, if you log every random request to the endpoint it could be a bit of a DoS vector, as the logging will impair the site or bring it down maybe

Reply:
Good call. Might be better if I remove it for now and bring it back via a `debug` option or something similar 🤔

Reply:
Removed this for now. Will re-add later via a `debug`-mode PR.